sickbeard/webapi.py

```diff
@@ -1054,7 +1054,7 @@ def convert_date(history_date):
                 ).strftime(dateTimeFormat)
 
             composite = Quality.splitCompositeStatus(cur_item.action)
-            if cur_type == statusStrings[composite.status].lower():
+            if cur_type is None or cur_type == statusStrings[composite.status].lower():
```

Comment:
Because, per the API if a type isn't selected then it should return all results.